editor/editor_node.cpp

```diff
@@ -3836,6 +3828,14 @@ void EditorNode::_set_current_scene_nocheck(int p_idx) {
 	if (tabs_to_close.is_empty()) {
 		callable_mp(this, &EditorNode::_set_main_scene_state).call_deferred(state, get_edited_scene()); // Do after everything else is done setting up.
```

Comment:
This comment might need rewording, as it's no longer the last action.

Reply:
no it is still done after everything because the call is deferred I think the commentary was about why it is deferred and not just a direct call and not about being at the end of the function